common/workunit/workunit.cpp

```diff
@@ -1087,6 +1089,17 @@ class CDaliWorkUnit : public CPersistedWorkUnit
         VStringBuffer path("/GraphProgress/%s/%s", queryWuid(), graphName);
         return querySDS().connect(path, myProcessSession(), RTM_LOCK_WRITE|RTM_CREATE_QUERY, SDS_LOCK_TIMEOUT);
     }
+    IPropertyTree *getGraphProgressTree() const
+    {
+        IRemoteConnection *conn = queryProgressConnection();
+        if (conn)
+        {
+            Owned<IPropertyTree> tmp = createPTree("GraphProgress");
+            mergePTree(tmp,conn->queryRoot());
```

Comment:
Potentially, does it matter and isn't the conn root names 'GraphProgress' anyway?